scripts/scripts.js

```diff
@@ -54,6 +54,12 @@ const CONFIG = {
     '--blog--adobecom.hlx.page': {
       'blog.adobe.com': 'origin',
     },
+    '--blog--adobecom.aem.live': {
+      'blog.adobe.com': 'origin',
+    },
+    '--blog--adobecom.aem.page': {
+      'blog.adobe.com': 'origin',
+    },
```

Comment:
Nit: we cam use regex here to reduce the verbosity
```suggestion
    '^https://.*--blog--adobecom.(hlx|aem).(page|live)': {
      'blog.adobe.com': 'origin',
    }
```

Reply:
P.S. You can also remove [these lines](https://github.com/adobecom/blog/pull/52/files#diff-fbceb58f32b99b8a5309251f8c49ac8eda82fba58b905c600c92bce1ed2a1c8cR51-R56) if you go the regex route.